README.md

```diff
@@ -1,40 +1,46 @@
-# cowin-notifier
 
-A cross platform tool which instantly notifies about COVID vaccine availability.
+
+# 💉 CoWIN Notifier 😷
+[![Open Source Love](https://badges.frapsoft.com/os/v1/open-source.svg?v=103)](https://github.com/aryan9600/cowin-notifier)
+[![GitHub License](https://img.shields.io/github/license/aryan9600/cowin-notifier)](https://github.com/aryan9600/cowin-notifier/blob/master/LICENSE)
+[![PRs Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg)](https://github.com/aryan9600/cowin-notifier/issues/new/choose)
+![release](https://img.shields.io/github/v/release/aryan9600/cowin-notifier)
+![rust-version](https://img.shields.io/badge/rust-v1.5.2-red)
+
+A cross-platform tool written in rust, which instantly notifies users about COVID-19 vaccine availability at their regions.
```

Comment:
I think it would be nice if we could mention this tool is compatible only in India